### PR TITLE
[LPT] Extending EliminateFakeQuantize transformation (two interval boundaries)

### PR DESCRIPTION
**Pull request by eshoguli**
### Details:
 - *Extending EliminateFakeQuantize transformation*

### Tickets:
 - *extending 98890 & workaround 107875*

### CODE REVIEW

File: src/common/low_precision_transformations/src/eliminate_fake_quantize.cpp [hunk 1]
```diff
@@ -46,25 +47,63 @@ bool EliminateFakeQuantizeTransformation::transform(TransformationContext& conte
...
+                    const std::shared_ptr<opset1::Constant>& constant,
+                    const float value,
+                    const float max_diff,
+                    const bool exact_comparison) noexcept {
+    bool need_to_check_intervals = false;
     const auto& constant_values = constant->cast_vector<float>();
     for (const auto constant_value : constant_values) {
         if (std::fabs(constant_value - value) > std::numeric_limits<float>::epsilon()) {
+            const auto diff = std::fabs(constant_value - value);
+            if ((exact_comparison && (std::fabs(constant_value - value) > std::numeric_limits<float>::epsilon())) ||
```
Comment (v-Golubev) [inline src/common/low_precision_transformations/src/eliminate_fake_quantize.cpp:60]: Please apply the changes from my [previous comment](https://github.com/openvinotoolkit/openvino/pull/17054#discussion_r1172323350)

File: src/common/low_precision_transformations/src/eliminate_fake_quantize.cpp [hunk 2]
```diff
@@ -46,25 +47,63 @@ bool EliminateFakeQuantizeTransformation::transform(TransformationContext& conte
...
+    }
+
+    if (need_to_check_intervals) {
+        auto tmp_fq = as_type_ptr<opset1::FakeQuantize>(fq->clone_with_new_inputs({
+            constant,
+            fq->get_input_node_shared_ptr(1),
+            fq->get_input_node_shared_ptr(2),
+            fq->get_input_node_shared_ptr(3),
+            fq->get_input_node_shared_ptr(4)}));
+        auto result = NetworkHelper::fold_fake_quantize(tmp_fq, false, 1);
```
Comment (v-Golubev) [inline src/common/low_precision_transformations/src/eliminate_fake_quantize.cpp:76]: These changes contain a conflict with #17096. Let's discuss offline in which order these PRs will be merged

**Review by v-Golubev — APPROVED** [review]
LGTM
BTW should we close #17054 as a duplicate?

File: src/common/low_precision_transformations/src/eliminate_fake_quantize.cpp (hunk 1 above)
Comment (eshoguli) [reply src/common/low_precision_transformations/src/eliminate_fake_quantize.cpp:60]: discussed offline, will be fixed in separate PR

**Pull request merged**